src/audio/manager.cpp

```diff
@@ -684,36 +483,36 @@ std::vector<ma_backend> AudioManager::ParseBackendsList(const Glib::ustring &lis
 
 #ifdef WITH_RNNOISE
 float AudioManager::GetCurrentVADProbability() const {
-    return m_vad_prob;
+    return m_voice->GetCapture().GetEffects().GetNoise().GetPeakMeter().GetPeak();
 }
 
 double AudioManager::GetRNNProbThreshold() const {
-    return m_prob_threshold;
+    return m_voice->GetCapture().GetEffects().GetNoise().m_vad_threshold;
```

Comment:
I see you reference the same thing later, I am gonna say this only here, but the same applies to all other occurrences.

I am not sure getters and setters make sense here, since they would effectively do the same thing, but the calls won't be able to get optimized away unless you build with LTO (which iirc is not the case for your release builds).

Reply:
i dont think optimizing away a function call is of any practical importance but if we wanna stick with it then we should call it something like `VADThreshold`. i just prefer get/set as a matter of style

Reply:
Yeah I am happy with that. Get/Set is ambiguous to the caller in what it would do, might just be a simple field or something more expensive.